docs/lint/commit.md

```diff
@@ -0,0 +1,26 @@
+## Commit Format Requirements
+1. The head line should contain no more than 72 characters
+2. head line is composed by  scope : subject 
+3. multi-scope is supported, separated by a space, such as scope1 scope2 scope3: subject 
+   it is better that scope number =<3 , scope length <= 20 char , but not mandatory
+4. keyword, such as  R4R  or WIP is not acceptable in the scope, no matter upper case or lower case.
```

Comment:
Does this PR contain this rule? r4r or wip is not acceptable in scope.

Reply:
added, please check